projects/tinkerbell/tink/docker/linux/tink-server/Dockerfile

```diff
@@ -1,9 +1,24 @@
 ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base
+ARG BUILDER_IMAGE
+
+FROM $BUILDER_IMAGE as wget-builder
+
+WORKDIR /newroot
+
+RUN set -x && \
```

Comment:
Can you run ldd on wget and verify the libraries it depends on are in the final image? Post the result here.

Reply:
I ran it during the builder stage
```
ldd ./usr/bin/wget
#15 2.712 	linux-vdso.so.1 (0x00007fff69ae7000)
#15 2.712 	libssl.so.10 => /lib64/libssl.so.10 (0x00007fe3cd0d9000)
#15 2.712 	libcrypto.so.10 => /lib64/libcrypto.so.10 (0x00007fe3ccc83000)
#15 2.712 	libz.so.1 => /lib64/libz.so.1 (0x00007fe3cca6e000)
#15 2.712 	libdl.so.2 => /lib64/libdl.so.2 (0x00007fe3cc86a000)
#15 2.712 	libidn.so.11 => not found
#15 2.712 	libuuid.so.1 => /lib64/libuuid.so.1 (0x00007fe3cc665000)
#15 2.712 	libpcre.so.1 => /lib64/libpcre.so.1 (0x00007fe3cc401000)
#15 2.712 	libc.so.6 => /lib64/libc.so.6 (0x00007fe3cc056000)
#15 2.712 	libgssapi_krb5.so.2 => /lib64/libgssapi_krb5.so.2 (0x00007fe3cbe0a000)
#15 2.712 	libkrb5.so.3 => /lib64/libkrb5.so.3 (0x00007fe3cbb26000)
#15 2.712 	libcom_err.so.2 => /lib64/libcom_err.so.2 (0x00007fe3cb922000)
#15 2.712 	libk5crypto.so.3 => /lib64/libk5crypto.so.3 (0x00007fe3cb6f1000)
#15 2.712 	/lib64/ld-linux-x86-64.so.2 (0x00007fe3cd348000)
#15 2.712 	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007fe3cb4d3000)
#15 2.712 	libkrb5support.so.0 => /lib64/libkrb5support.so.0 (0x00007fe3cb2c4000)
#15 2.712 	libkeyutils.so.1 => /lib64/libkeyutils.so.1 (0x00007fe3cb0c0000)
#15 2.712 	libresolv.so.2 => /lib64/libresolv.so.2 (0x00007fe3caeaa000)
#15 2.712 	libselinux.so.1 => /lib64/libselinux.so.1 (0x00007fe3cac83000)
```

libidn I am actually removing because i read it was listed as [optional](https://www.linuxfromscratch.org/blfs/view/svn/basicnet/wget.html). But I can add it back in 👍 

I also did a run on the final image and `wget --help` seemed to work fine.

Reply:
I'd probably keep it just in case. Since it shows up as a dynamic dep I'd be worried about not having it.

Reply:
Sure! I can add it back 👍